op-e2e/e2eutils/disputegame/honest_helper.go

```diff
@@ -38,7 +38,7 @@ func (h *HonestHelper) Defend(ctx context.Context, claimIdx int64) {
 	pos := types.NewPositionFromGIndex(claim.Position.Uint64())
 	defendPos := pos.Defend()
 	traceIdx := defendPos.TraceIndex(int(h.game.MaxDepth(ctx)))
-	value, err := h.correctTrace.Get(ctx, traceIdx)
+	value, err := h.correctTrace.Get(ctx, defendPos)
 	h.game.require.NoErrorf(err, "Get correct claim at trace index %v", traceIdx)
```

Comment:
I'd remove the traceIdx reference from these errors and include the position instead. Otherwise you'll get odd error messages when the trace provider is composing two different trace providers into a single game.